src/mock/data/brukere.ts

```diff
@@ -35,15 +35,41 @@ const lagTelefonnummer = (): string => {
 	return faker.phone.phoneNumber().replaceAll(' ', '')
 }
 
+const generateSluttDato = (status: TiltakDeltakerStatus, startDato: Date | null) =>  {
+	if(status === 'VENTER_PA_OPPSTART') {
+		return  null
+	} else if(status === 'HAR_SLUTTET' && startDato !== null) {
+		return  faker.date.between(startDato, Date())
+	} else {
+		return  faker.date.future()
+	}
+}
+
+const getStatus = (): TiltakDeltakerStatus => {
+	const i = randBetween(0, 10)
+
+	if(i < 5) return TiltakDeltakerStatus.DELTAR
+	if(i < 7) return TiltakDeltakerStatus.VENTER_PA_OPPSTART
+	if(i < 8) return TiltakDeltakerStatus.HAR_SLUTTET
+	if(i < 9) return TiltakDeltakerStatus.IKKE_AKTUELL
+
+	return TiltakDeltakerStatus.DELTAR
+
+}
+
 const lagMockTiltakDeltagerForGjennomforing = (gjennomforing: Gjennomforing): TiltakDeltakerDetaljer => {
-	const status = faker.random.objectElement(TiltakDeltakerStatus) as TiltakDeltakerStatus
+	const status = getStatus()
 
-	const brukerFornavn = faker.name.firstName()
-	const brukerMellomnavn = randBetween(0, 10) > 6 ? faker.name.middleName() : null
+	const gender = randBetween(0, 1)
+
+	const brukerFornavn = faker.name.firstName(gender)
+	const brukerMellomnavn = randBetween(0, 10) > 6 ? faker.name.firstName(gender) : null
 	const brukerEtternavn = faker.name.lastName()
 
 	const veilederNavn = faker.name.firstName() + ' ' + faker.name.lastName()
 
+	const startDato = status !== 'VENTER_PA_OPPSTART' ? faker.date.past() : null
```

Comment:
Her burde du bruke enumen isteden for å hardkode

Reply:
Venter på oppstart kan også ha startdato i fremtiden, ikke sikkert vi skal legge det inn da